packages/devtools_app/lib/src/network/flutter/http_request_inspector_views.dart

```diff
@@ -41,9 +43,7 @@ class HttpRequestHeadersView extends StatelessWidget {
         children: [
           Text(
             '$key: ',
-            style: const TextStyle(
-              fontWeight: FontWeight.bold,
-            ),
+            style: Theme.of(context).textTheme.subtitle,
```

Comment:
if you are passing around `context` so you can access textTheme, just pass the textTheme instead